hawkular-wildfly-agent-feature-pack/src/main/resources/modules/system/layers/base/org/hawkular/agent/main/module.xml

```diff
@@ -83,6 +83,7 @@
     <module name="org.jboss.vfs"/>
 
     <!-- other dependencies we need -->
+    <module name="javaee.api"/> <!-- bus-commons -->
```

Comment:
Does bus-commons really need JEE? I think that would be bad because it means we won't be able to install the agent in any WildFly-core server or any server that doesn't have JEE (like a trimmed down swarm instance).